selfdrive/car/hyundai/values.py

```diff
@@ -225,7 +225,10 @@ def init_make(self, CP: car.CarParams):
   ],
 
   # Genesis
-  CAR.GENESIS_GV60_EV_1ST_GEN: HyundaiCarInfo("Genesis GV60 2023", "All", harness=Harness.hyundai_k),
+  CAR.GENESIS_GV60_EV_1ST_GEN: [
+    HyundaiCarInfo("Genesis GV60 (Advanced Trim) 2023", "All", harness=Harness.hyundai_a),
+    HyundaiCarInfo("Genesis GV60 (Performance Trim) 2023", "All", harness=Harness.hyundai_k),
```

Comment:
difference isn't HDA?

Reply:
See nothing about it in the brochure, and neither dongles have an ADAS ECU